src/printer/tvmscript_printer.cc

```diff
@@ -457,6 +457,12 @@ class TVMScriptPrinter : public StmtFunctor<Doc(const Stmt&)>,
  */
 template <typename T>
 void NDArrayToTIR(::tvm::runtime::NDArray arr, std::ostream& os) {
+  if (arr.DataType().code() == runtime::DataType::kInt ||
+      arr.DataType().code() == runtime::DataType::kUInt) {
+    // Printing integer NDArrays causes "UnicodeDecodeError: 'utf-8' codec can't decode byte"
+    // error during MetaSchedule tuning on int8 models.
+    return;
+  }
```

Comment:
We can't print an entire constant tensor to text, so this function is broken anyway in practice. I've update one of TVMScript parsing tests to avoid printing integer constants.